### DISCUSSION TITLE
How to request to join Github organization?

### DISCUSSION THREAD

**Discussion by multikitty** [community/community#53914] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi, everyone!
I'd like to join Github organization.
Who can help me?

**Comment by multikitty** [comment]
thanks, but I can't see Join button.

**Comment by brucebanner1122** [comment]
To request to join a GitHub organization, follow these simple steps:  

1. Find the Organization – Go to the GitHub page of the organization you want to join.  
2. Check for an Invite Option – Some organizations allow you to request to join directly. If there’s a “Request to Join” button, click it!  
3. Reach Out to an Admin – If there’s no option to request, you’ll need an invite. Contact an admin or someone in the organization and ask them to send you an invite.  
4. Accept the Invitation – Once they send an invite, you’ll get a notification in your GitHub email or on your GitHub dashboard. Click “Accept,” and you’re in!  

If you're unsure who to contact, check the organization’s website or GitHub page for guidelines. 🚀
